backend/src/apps/owasp/models/activity_event.py

```
@@ -0,0 +1,153 @@
+        return [(ActivityEvent.ActivityType.RELEASE_PUBLISHED, occurred_at, release.author)]
+
+    @staticmethod
+    def update_data(obj) -> None:
+        """Create ActivityEvent row(s) for a saved GitHub model instance if they do not exist."""
+        handler_name = ActivityEvent.HANDLERS.get(type(obj).__name__)
+        if handler_name is None:
+            logger.error(
+                "ActivityEvent.update_data received unsupported model type: %s",
+                type(obj).__name__,
+            )
+            raise TypeError(f"Unsupported model type: {type(obj)}")
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)